sdp/processors/modify_manifest/data_to_data.py

```diff
@@ -14,13 +14,232 @@
 
 import collections
 import re
+import os
 from typing import Dict, List
+import soundfile as sf
 
 from sdp.logging import logger
 from sdp.processors.base_processor import BaseParallelProcessor, DataEntry
+from sdp.utils.common import ffmpeg_convert
 from sdp.utils.edit_spaces import add_start_end_spaces, remove_extra_spaces
 from sdp.utils.get_diff import get_diff_with_subs_grouped
 
+    
+class AudioDuration(BaseParallelProcessor):
+    """
+    Count audio duration using audio file path from input_field
+
+    Args:
```

Comment:
maybe need more descriptive arg names. `audio_filepath_field` and `duration_field` ?

Reply:
renamed